### DISCUSSION TITLE
pb authentification github

### DISCUSSION THREAD

**Discussion by eziopelle** [community/community#73597] [New to GitHub]
Bonjour, 

J'ai un problème quand je veux pousser mon travail sur un nouveau repo github : 

le message d'erreur est le suivant : 

➜  c-troop-appli git:(master) ✗ git push origin master
Username for 'https://github.com': eziopelle
Password for 'https://eziopelle@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/eziopelle/c-troop-appli/'

Le repo est bien créé pourtant, je ne comprends pas trop. 

Quelqu'un peux m'aider svp ?

Très bonne journée,

**Comment by airtower-luna** [comment]
You cannot use your password for `git push` or other Git operations on GitHub (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Reply by queenofcorgis** [reply]
@airtower-luna happy to see you back around these parts :cowboy_hat_face: I'd love to catch up more when you get a moment :heart:
